FitNesseRoot/FitNesse/SuiteAcceptanceTests/SuiteSlimTests/TestSystemSlimSuite/ErrorSituationsDuringSlimServerStartup/A_TimeoutAsSlimHeaderNotReceived/content.txt

```diff
@@ -19,5 +19,5 @@ After slim.timeout seconds we should abort and stop the test.
 |ensure        |content contains|Timeout while reading slim header from client.|
 |show collapsed|content                                                        |
 |check         |request page    |TestPage?executionLog           |200          |
-|ensure        |content contains|Address already in use                        |
+|ensure        |content contains|java.net.BindException                        |
 |show collapsed|content                                                        |
```

Comment:
Can you also check for `(Bind failed)`, which I gather from your report is also present in German

Reply:
I changed it to match for both versions, so that the original exception message is still visible in the test. That way it's easier to understand what it's doing.